troposphere/codebuild.py

```diff
@@ -111,6 +111,7 @@ def validate(self):
             valid_types = [
                 'PARAMETER_STORE',
                 'PLAINTEXT',
+                'SECRETS_MANAGER'
```

Comment:
Ah, just realized this is already merged in:

https://github.com/cloudtools/troposphere/blob/1ca9e8c0ef00fa5cf333aae038c7f362855e6cfb/troposphere/codebuild.py#L114

nevermind then